### DISCUSSION TITLE
Which version to use for imminent greenfield project?

### DISCUSSION THREAD

**Discussion by earslap** [statelyai/xstate#4455] [General] [answered]
I am familiar with the "it's done when it is done" aspect of things, but can the team provide any guidance? I think things are a bit in a limbo at the moment, v5 is always soon-ish, and I saw some different tentative release dates scattered around all of which has been passed. I see that there are still moving parts with v5, there are gaps / "Coming soon..."s in the documentation etc. The website says v4 visualiser / editor is not being maintained. So I'm assuming resources are devoted to v5 but what can we expect throughout the road to v5 release? Is there a realistic timeline that can be shared?

**Comment by davidkpiano** [comment, marked as answer]
XState v5 will be released by the end of November.

**Reply by ovidb** [reply]
Music to my ears!
